packages/block-editor/src/components/link-control/search-results.js

```diff
@@ -53,25 +52,16 @@ export default function LinkControlSearchResults( {
 	// See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role
 	const searchResultsLabelId = `block-editor-link-control-search-results-label-${ instanceId }`;
 	const labelText = isInitialSuggestions
-		? __( 'Recently updated' )
+		? __( 'Links' )
```

Comment:
Given that this is not visible should we be more descriptive here? 
```suggestion
		? __( 'Search results' )
```

Reply:
Perhaps "Link suggestions"? Or just "Suggestions".

Reply:
I don't think this should block the effort; perhaps worth a follow-up as we get suggestions surfaced holistically with https://github.com/WordPress/gutenberg/issues/50884.

Reply:
> Also changes the text to "Links" instead of "Recently updated" as the results consists of all sorts of links - with varying defaults.

Can we have the title match the result of the lists? If not, I like "Suggestions" as it's general and feels contextual without being specific and lets people know this is not the result of their search, but defaults that were recommended.

After the search, having it be "Search Results" is descriptive and short. I don't think "Links" is very helpful information in this case.

Reply:
The "Search results" title is already handled in code beneath that. I think changing "Links" here to "Suggestions" is best.